docs/installing.rst

```diff
@@ -148,8 +148,22 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+---------------------------------------------------
+3. Install dependencies for vagrant-service-manager
+---------------------------------------------------
+
+For some operating systems, you might need to install additional dependencies.
+
+* Fedora 24
+
+  Run the following commands to install ``Development Tools``, ``rpm-build``, and ``zlib-devel`` packages::
+
+  $ sudo dnf install "Development Tools"
+  $ sudo dnf install rpm-build
+  $ sudo dnf install zlib-devel
+
 -----------------------------------------------------
-3. Install the vagrant-service-manager Vagrant plugin
+4. Install the vagrant-service-manager Vagrant plugin
```

Comment:
We now require vagrant-sshfs as well, I believe it is appropriate to add that in this PR